provider/src/main/java/au/com/dius/pact/provider/junitsupport/loader/VersionSelector.java

```diff
@@ -0,0 +1,25 @@
+package au.com.dius.pact.provider.junitsupport.loader;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Used to specify which versions to use when querying the Pact matrix.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@Inherited
+public @interface VersionSelector {
+    /**
+     * @return Tags to use to fetch pacts for, defaults to `latest`
+     */
+    String tag() default "latest";
```

Comment:
I don't think latest here is right. It probably should be empty or mandatory.

Reply:
Removed the default and made it mandatory.